lost/src/main/java/com/mapzen/android/lost/internal/GeofencingApiImpl.java

```diff
@@ -70,12 +72,16 @@ public void disconnect() {
   private PendingResult<Status> addGeofence(LostApiClient client, Geofence geofence,
       PendingIntent pendingIntent) throws SecurityException {
 
-    internalIntent = intentFactory.createIntent();
-    internalIntent.putExtra(GeofencingIntentService.EXTRA_PENDING_INTENT, pendingIntent);
+    int pendingIntentId = (int) System.currentTimeMillis();
+    internalIntent = intentFactory.createIntent(context);
+    internalIntent.addCategory(String.valueOf(pendingIntentId));
```

Comment:
When we added extras to the `internalIntent`, the "entering" extra that the `LocationManager` was supposed to add wasn't present when the intent fired :/